packages/synapse-react-client/src/mocks/msw/handlers/accessRequirementHandlers.ts

```diff
@@ -84,43 +90,110 @@ export const getAccessRequirementEntityBindingHandlers = (
+    },
+  )
+
+export const getAccessRequirementStatusHandlers = (
+  backendOrigin: string,
+  arStatusOverrides?: AccessRequirementStatus[],
+) => {
+  if (arStatusOverrides) {
+    arStatusOverrides.forEach(override => {
+      accessRequirementStatuses.set(override.accessRequirementId, override)
+    })
+  }
+  return [
+    rest.get(
+      `${backendOrigin}${ACCESS_REQUIREMENT_STATUS(':id')}`,
+
+      async (req, res, ctx) => {
+        let response: AccessRequirementStatus | undefined
+        const accessRequirement = mockAccessRequirements.find(
+          accessRequirement =>
+            req.params.id === accessRequirement.id.toString(),
+        )
+        let override = accessRequirementStatuses.get(req.params.id as string)
+        if (override) {
+          response = override
+        }
+        if (!response && accessRequirement) {
+          const isManagedACTAR =
+            accessRequirement.concreteType ===
+            MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
+          response = {
+            accessRequirementId: req.params.id as string,
+            concreteType: isManagedACTAR
+              ? 'org.sagebionetworks.repo.model.dataaccess.ManagedACTAccessRequirementStatus'
+              : 'org.sagebionetworks.repo.model.dataaccess.BasicAccessRequirementStatus',
+            isApproved: true,
+            currentSubmissionStatus: isManagedACTAR
+              ? {
+                  submissionId: mockApprovedSubmission.id,
+                  submittedBy: mockApprovedSubmission.submittedBy,
+                  modifiedOn: mockApprovedSubmission.modifiedOn,
+                  state: SubmissionState.APPROVED,
+                }
+              : undefined,
+          }
         }
+        const status = response ? 200 : 404
+        return res(ctx.status(status), ctx.json(response))
+      },
+    ),
+  ]
+}
+
+export function getCreateAccessApprovalHandler(backendOrigin: string) {
+  return rest.post(
+    `${backendOrigin}/repo/v1/accessApproval`,
+
+    async (req, res, ctx) => {
+      const requestBody: CreateAccessApprovalRequest = await req.json()
+      const status = 200
+      const response: AccessApproval = {
+        ...requestBody,
+        id: 123,
+        etag: 'mock-etag',
+        createdOn: new Date().toISOString(),
+        modifiedOn: new Date().toISOString(),
+        createdBy: String(MOCK_USER_ID),
+        modifiedBy: String(MOCK_USER_ID),
       }
-      const status = response ? 200 : 404
+
+      // Update the corresponding status object
+      // TODO: Status should be inferred from the presence of an access approval, rather than the other way around
+      accessRequirementStatuses.set(String(requestBody.requirementId), {
+        accessRequirementId: String(requestBody.requirementId),
+        concreteType:
+          'org.sagebionetworks.repo.model.dataaccess.BasicAccessRequirementStatus',
+        isApproved: true,
+      })
+
```

Comment:
Update the map so MSW will now return that the corresponding AccessRequirementStatus is approved.

Add a note because we should probably store the mock AccessApproval and infer the status based on the presence of the mock AccessApproval object.